recipes/namd/3.0b6/gh200/environments.yaml

```diff
@@ -0,0 +1,15 @@
+namd:
+  compiler:
+  - toolchain: gcc
+    spec: gcc@12.3
+  unify: true
+  specs:
+  - cuda@12
+  - fftw@3.3.10 +openmp ~mpi
+  - charmpp@7 backend=multicore +production +tracing
+  - namd@3.0b6 +cuda cuda_arch=90 +single_node_gpu
+  views:
+    namd-dev:
+      exclude: ["namd"]
```

Comment:
Following our best practices, only include `root` packages in the view:
```yaml
    link: roots
```
This avoids some linking and software compatibility issues.

When using `roots`, you have to make sure that all required dependencies are provided as root specs. This might require some iteration. Make notes about the sequence of commands that you used to get / untar the source code, configure the env, configure the build, and build - and add them to the docs for NAMD

Reply:
I added all the dependencies explicitly and instructions on how to build from source (using the bounded version of Charm++).